.github/workflows/ci.yml

```
@@ -70,11 +70,14 @@ jobs:
       - name: Init cache
         uses: Swatinem/rust-cache@v2
 
-      - name: Install Fuel toolchain
-        uses: FuelLabs/action-fuel-toolchain@v0.6.0
+      - name: Install fuel toolchain
+        uses: cachix/install-nix-action@v21
         with:
-          toolchain: latest
-          date: 2023-04-12
+          nix_path: nixpkgs=channel:nixos-unstable
+          extra_nix_config: |
+            extra-substituters = https://fuellabs.cachix.org
+            extra-trusted-public-keys = fuellabs.cachix.org-1:3gOmll82VDbT7EggylzOVJ6dr0jgPVU/KMN6+Kf8qx8=
+      - run: nix shell github:fuellabs/fuel.nix#forc-0-39-0
```

This should install it globally so the other CI steps have access to forc

```suggestion
      - run: nix profile install github:fuellabs/fuel.nix#forc-0-39-0
```